trimmerdecorator.rb

```diff
@@ -0,0 +1,5 @@
+class TrimmerDecorator < Decorator
```

Comment:
* Since you are inheriting from class `Decorator` kindly require the decorator class in here 👍